rampuptime/ramp_up_time.go

```diff
@@ -2,37 +2,87 @@ package rampuptime
 
 import (
 	"fmt"
+	"net/http"
+	"encoding/json"
 	"regexp"
-
-	"github.com/KevinMi2023p/ECE461_TEAM33/npm"
+	. "github.com/KevinMi2023p/ECE461_TEAM33/common"
+	. "github.com/KevinMi2023p/ECE461_TEAM33/npm"
 )
 
-//This module uses the "npm" package that's created locally
+const github_readme_url_part string = "%s/readme"
 
-func Ramp_up_score(json_data *npm.NpmInfo) float32 {
+//This module uses the "npm" package that's created locally
+func Ramp_up_score_npm(json_data *NpmInfo) float32 {
 	//Below is a link of what <json_data> should be
 	//https://github.com/npm/registry/blob/master/docs/responses/package-metadata.md
 
 	if json_data == nil {
-		fmt.Println("NPM package hasn't been called yet or doesn't exist")
+		// fmt.Println("NPM package hasn't been called yet or doesn't exist")
 		return 0
 	}
 
 	//Look through map to get README data
-	var readme_string = npm.Get_nested_value_from_info(json_data, []string{"readme"})
+	var readme_string = Get_nested_value_from_info(json_data, []string{"readme"})
```

Comment:
without "npm." how does this not syntax error, since that function is from the "npm" compartment.